test/vstest.console.PlatformTests/AssemblyMetadataProviderTests.cs

```diff
@@ -75,9 +75,11 @@ public void GetArchitectureForNativeDll(string platform)
             var arch = this.assemblyMetadataProvider.GetArchitecture(assemblyPath);
             stopWatch.Stop();
 
-            Assert.AreEqual(Enum.Parse(typeof(Architecture), platform, ignoreCase: true), arch);
             Console.WriteLine("Platform:{0}, {1}", platform, string.Format(PerfAssertMessageFormat, expectedElapsedTime, stopWatch.ElapsedMilliseconds));
-            Assert.IsTrue(stopWatch.ElapsedMilliseconds < expectedElapsedTime, string.Format(PerfAssertMessageFormat, expectedElapsedTime, stopWatch.ElapsedMilliseconds));
+            Assert.AreEqual(Enum.Parse(typeof(Architecture), platform, ignoreCase: true), arch);
+
+            // We should not assert on time elapsed, it will vary depending on machine, & their state, commenting below assert
+            // Assert.IsTrue(stopWatch.ElapsedMilliseconds < expectedElapsedTime, string.Format(PerfAssertMessageFormat, expectedElapsedTime, stopWatch.ElapsedMilliseconds));
```

Comment:
can we remove it then ?

Reply:
I don't want to remove this commented out as:

1. It is in test code so I can live with it.
2. If I had removed it to being with, I would have been able to explain why I did so in first place.
3. It's kind of a reminder that do not add perf test in Unit Tests